docs/source/index.rst

```diff
@@ -396,7 +396,8 @@ Polling is not subject to handle state with HTTP status codes (e.g. 304)
           "calamari": false,
           "conf": {"global": {"auth supported": "cephx"}},
           "host": "mon1.example.com",
-          "monitor_interface": "eth0",
+          "interface": "eth0",
+          "address": "10.0.0.1",
```

Comment:
we should not give examples using both because that is not an ideal scenario. Using `interface` by default is fine as an example. Of course we should explain that either-or acceptance